google/cloud/aiplatform/jobs.py

```diff
@@ -420,7 +433,11 @@ def create(
                 f"type. Please choose from: {constants.BATCH_PREDICTION_OUTPUT_STORAGE_FORMATS}"
             )
 
-        gapic_batch_prediction_job = gca_bp_job.BatchPredictionJob()
+        gapic_batch_prediction_job = gca_bp_job.BatchPredictionJob(
+            encryption_spec=initializer.global_config.get_encryption_spec(
```

Comment:
We may want to mutate `gapic_batch_prediction_job` instead of passing in the encryption spec into the constructor to be consistent with the rest of the composition below. Or compose it all through the constructor.

Reply:
Done